ansible/roles/postgresql/tasks/main.yml

```diff
@@ -180,6 +180,16 @@
   tags:
     - pgbouncer
 
+- shell: /bin/true
+  register: no_tags
```

Comment:
This is a nice approach! Are there other things that need to happen once, that we can use this with too?